### DISCUSSION TITLE
When adding a new time slot to a date, it should start after the end of the previous one

### DISCUSSION THREAD

**Discussion by amandasystems** [lukevella/rallly#138] [Ideas]
The current behaviour when adding a time slot is to have the new slot start at the same time as the previous one, which is usually never what you want. It would be better to have it start after the end of the previous one.

Of course, this might have weird edge cases, possibly interacting with the split between time slots and days (what happens if the previous slot ends at midnight?).

I volunteer to implement this since I believe it would be a good warm-up exercise for me.

**Comment by lukevella** [comment]
Hey there, thanks for your suggestion. Sounds reasonable, though I have some rework planned in this area so I will need to hold off on this for now as I'm not sure this will be relevant after the update I have planned.
